scripts/stabilization.py

```diff
@@ -118,6 +118,13 @@ class StabilizationParameter : public Expression
     double Dh_x;
     double max_Dh = 0;
     
+    // find smallest edge on cell
+    if (dim > 1) {
+        for(EdgeIterator edge(cell); !edge.end(); ++edge) {
+            h = std::min(h, edge->length());
+        }
+    }
```

Comment:
Well, this is a reasonable way of computing `h`, but not the way it is usually defining in the finite element context. It would interessting how big the difference to `cell.h()`is.